src/components/EmailList/index.jsx

```diff
@@ -0,0 +1,125 @@
+import EmailItem from "../EmailItem";
+import { useEffect, useState } from "react"
+import EmailView from "../EmailView";
+
+function EmailList() {
+
+    const [emailArray, setEmailArray] = useState(false);
+    const [emailID, setEmailID] = useState(0);
+    const [emailRead, setEmailRead] = useState(0);
+    const [refreshJson, setRefreshJson] = useState(false);
+
+    //To pass into email view
+    const [emailAddress, setEmailAddress] = useState("");
+    const [emailSubject, setEmailSubject] = useState("");
+    const [emailName, setEmailName] = useState("");
+    const [emailDate, setEmailDate] = useState("");
+    const [emailBody, setEmailBody] = useState("");
+
+    const [emailData, setEmailData] = useState([]);
+    let eID = null;
+
+    useEffect(() => {
+        console.log("email ID'd");
+        eID = emailID;
+    }, [emailID]);
+
+    useEffect(() => {
+        fetch("https://email-client-api.dev.io-academy.uk/emails")
+        .then(response => response.json())
+        .then(data => {
+            console.log(data);
+            setEmailArray(data);
+            setRefreshJson(false);
+            return(
+                emailArray
+            )
+        })
+    }, [refreshJson])
+
+
+
+    //SETS EMAIL TO READ
```

Comment:
Don't need this comment - the function is called `setRead`, so I would really hope that it sets emails to read!